example/java/TransactionExample.java

```diff
@@ -0,0 +1,163 @@
+import iroha.protocol.Commands;
+import iroha.protocol.BlockOuterClass;
+import iroha.protocol.Endpoint;
+import iroha.protocol.Queries;
+import iroha.protocol.Queries.Query;
+import iroha.protocol.Queries.GetAssetInfo;
+
+import iroha.protocol.QueryServiceGrpc;
+import iroha.protocol.QueryServiceGrpc.QueryServiceBlockingStub;
+import iroha.protocol.CommandServiceGrpc;
+import iroha.protocol.CommandServiceGrpc.CommandServiceBlockingStub;
+import iroha.protocol.Endpoint.TxStatus;
+import iroha.protocol.Endpoint.TxStatusRequest;
+import iroha.protocol.Endpoint.ToriiResponse;
+import iroha.protocol.Responses.QueryResponse;
+import iroha.protocol.Responses.AssetResponse;
+import iroha.protocol.Responses.Asset;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.Descriptors.FieldDescriptor;
+
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.lang.Thread;
+
+class TransactionExample {
+    static {
+        try {
+            System.loadLibrary("irohajava");
+        } catch (UnsatisfiedLinkError e) {
+            System.err.println("Native code library failed to load. \n" + e);
+            System.exit(1);
+        }
+    }
+
+    private static ModelCrypto crypto = new ModelCrypto();
+    private static ModelTransactionBuilder txBuilder = new ModelTransactionBuilder();
+    private static ModelQueryBuilder queryBuilder = new ModelQueryBuilder();
+    private static ModelProtoTransaction protoTxHelper = new ModelProtoTransaction();
+    private static ModelProtoQuery protoQueryHelper = new ModelProtoQuery();
+
+
+
+    public static void main(String[] args) {
```

Comment:
Would be nice to extract functions from main. E.g `readKeyFrom`, `bytesFromVector`, etc